### PR TITLE
Document deps management, with tooling installation

### PR DESCRIPTION
**Pull request by lalyos**

### CODE REVIEW

File: DEPS.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+- [glide-vc](github.com/sgotti/glide-vc) vendor cleaner plugin for glide
+
+`make deps` will instal them, in case they are missing
+
+## Add a new dependency
+
+If you write new features which imports a new library, you have to vendor it:
+```
+glide get -v github.com/Masterminds/cookoo/web
+glide-vc --only-code
```
Comment (matyix) [inline DEPS.md]: Shall we not add `--no-tests` as well to remove tests from vendor?

File: DEPS.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+```
+- package: github.com/prometheus/prometheus
+  version: ^2.0.0
+```
+
+than perform an update:
+
+```
+glide up -v
+glide-vc --only-code
```
Comment (matyix) [inline DEPS.md]: Same here re `--no-tests`?

File: DEPS.md (hunk 1 above)
Comment (lalyos) [reply DEPS.md]: good catch

File: DEPS.md (hunk 2 above)
Comment (lalyos) [reply DEPS.md]: same here

**Pull request merged**